build.gradle

```diff
@@ -117,6 +117,7 @@ dependencies {
     compile("ch.qos.logback.contrib:logback-json-classic:0.1.5")
     compile("ch.qos.logback.contrib:logback-jackson:0.1.5")
     compile("net.logstash.logback:logstash-logback-encoder:5.2")
+    compile("com.auth0:java-jwt:3.10.3")
```

Comment:
cx-flow uses Java Cryptography Extension (JCE) and not Java JWT (JSON Web Token).
there is no point in adding another package that does the same thing.